ietf/static_src/css/iab.scss

```diff
@@ -56,16 +65,28 @@
 }
 
 .iab-logo-image {
-    width: 100px !important;
+    width: 50px !important;
+
+    @media (min-width: 440px) {
```

Comment:
This will work but it's not the best practice as we're using hardcoded numbers here rather than a variable. Also adding a breakpoint into the mix isn't great for maintainability.

A better approach would be to override bootstraps base breakpoints. From the project setup I think you may be able to add this into the BS-override.scss (https://github.com/springload/wagtail_website/blob/integration/ietf/static_src/css/bs-override.scss)

In here you should be able to add:
```
$grid-breakpoints: (
  xs: 0,
  sm: 576px,
  md: 768px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px
);
```
And switch out the `sm` breakpoint for 440px. It is used in a few other places but this should be fine/fix more things they haven't noticed.

The import order may be tricky with the override but let me know if it works or not and can go from there.

Reply:
Looks like it works fine, those changes have been pushed